From: Johannes Willkomm <j.willkomm@fionec.de>
Date: Fri, 12 Aug 2022 16:18:24 +0200
Subject: Fix makefile

---
 Makefile | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 5a94905..78a3b2a 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,12 @@ install:
 	@echo PWD=$(shell pwd)
 	mkdir -p $(DESTDIR)$(PREFIX)
 	rm -rf $(DESTDIR)$(PREFIX)/$(PKGNAME)
-	git clone $(shell pwd) $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	mkdir -p $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd)/ci $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd)/data $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd)/doc $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd)/fontcache $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd)/xc $(DESTDIR)$(PREFIX)/$(PKGNAME)
 
 configure-system: install
 	$(DESTDIR)$(PREFIX)/$(PKGNAME)/ci/config-system.sh
