From: Johannes Willkomm <j.willkomm@fionec.de>
Date: Thu, 7 Jul 2022 13:13:27 +0200
Subject: Use copy instead of git clone for install

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7de0779..d5c1f9f 100644
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ install:
 	@echo PWD=$(shell pwd)
 	mkdir -p $(DESTDIR)$(PREFIX)
 	rm -rf $(DESTDIR)$(PREFIX)/$(PKGNAME)
-	git clone $(shell pwd) $(DESTDIR)$(PREFIX)/$(PKGNAME)
+	cp -ra $(shell pwd) $(DESTDIR)$(PREFIX)/$(PKGNAME)
 
 configure-system:
 	$(DESTDIR)$(PREFIX)/$(PKGNAME)/ci/config-system.sh
