From: Johannes Willkomm <j.willkomm@fionec.de>
Date: Thu, 18 Aug 2022 12:10:56 +0200
Subject: Bugfix: do not try to convert '' to int

---
 xc/main/views.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xc/main/views.py b/xc/main/views.py
index c1dada2..ac88748 100644
--- a/xc/main/views.py
+++ b/xc/main/views.py
@@ -1895,7 +1895,7 @@ def ajax_action(request):
             elif lsl['info']['exec'] == 0:
                 errmsg = 'File is not executable'
 
-            elif int(cdata['nowait']) == 1:
+            elif len(cdata['nowait']) > 0 and int(cdata['nowait']) == 1:
                 result = workdir.execbg([workdir.realpath(path)],  {'user': request.user.username, 'comment': cdata['comment']})
                 print('xc started process async:', result)
                 return redirect(next_)
