From: Johannes Willkomm <j.willkomm@fionec.de>
Date: Thu, 18 Aug 2022 14:09:24 +0200
Subject: Another small bug regarding nowait

---
 xc/main/views.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xc/main/views.py b/xc/main/views.py
index ac88748..a69c263 100644
--- a/xc/main/views.py
+++ b/xc/main/views.py
@@ -1842,7 +1842,7 @@ class ActionData(XCForm):
     path = forms.CharField(max_length=1024, label='Path')
     next_ = forms.CharField(required=False, max_length=1024, label='Follow-up action')
     comment = forms.CharField(required=False, max_length=1024, label='Comment', widget=forms.Textarea)
-    nowait = forms.ChoiceField(required=False, label='NoWait', choices=[(0,0),(1,1)])
+    nowait = forms.ChoiceField(required=False, label='NoWait', choices=[('0',0),('1',1)])
 
     def clean(self):
         cleaned_data = super().clean()
