From: Johannes Willkomm <j.willkomm@fionec.de>
Date: Tue, 9 Aug 2022 16:17:22 +0200
Subject: Remove nginx default site on install

---
 ci/config-system.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ci/config-system.sh b/ci/config-system.sh
index a2bd842..9ed17de 100755
--- a/ci/config-system.sh
+++ b/ci/config-system.sh
@@ -52,6 +52,10 @@ sed -e "s
 
 ln -sfT $mydir/xc_nginx.conf /etc/nginx/sites-available/xc_nginx.conf
 
+if [[ -f /etc/nginx/sites-enabled/default ]]; then
+    rm /etc/nginx/sites-enabled/default
+fi
+
 cp $mydir/ci/uwsgi_params $mydir/
 
 cd /etc/nginx/sites-enabled && ln -sf ../sites-available/xc_nginx.conf
